replica/compaction_group.hh

```diff
@@ -260,9 +260,17 @@ public:
     // FIXME: Cannot return nullptr, signature can be changed to return storage_group&.
     storage_group* storage_group_for_id(const schema_ptr&, size_t i) const;
 
-    // Caller must keep the current effective_replication_map_ptr valid
-    // until the storage_group_manager finishes update_effective_replication_map
-    virtual future<> update_effective_replication_map(const locator::effective_replication_map& erm) = 0;
+    class erm_updater {
```

Comment:
Please avoid obscure acronyms in public interfaces?

Reply:
sure.